web/src/features/AppVersionHistory/AppVersionHistoryRow.jsx

```diff
@@ -34,33 +36,6 @@ class AppVersionHistoryRow extends Component {
     );
   };
 
-  renderYamlErrors = (version) => {
-    if (!version.yamlErrors) {
-      return null;
-    }
```

Comment:
I kept this in the parent component. In general I don't think components should ever return null- their parents should determine if null should be rendered.